cmd/protoc-gen-cpp-tableau-loader/embed/logger.pc.h

```diff
@@ -54,6 +54,8 @@ class Logger {
   void SetWriter(Writer writer) { writer_ = writer; }
   // Log with guarantee of thread-safety.
   void Log(const SourceLocation& loc, Level level, const char* format, ...);
+  // Mkdir recursively
+  int Mkdir(std::string path);
```

Comment:
This is a utility function, and should be placed at namespace `tableau::util`: 
https://github.com/tableauio/loader/blob/653080fb2ca6772d123d015c2a09af88b9ec1f3a/_lab/cpp/src/demo/hub.pc.h#L173

Reply:
> * `Mkdir` modifies the input string to make logic correct, so just pass the string as value.
> 
> ```c++
> int Mkdir(std::string path) {
>   path += "/";
>   ...
> }
> ```
> 
Not good idea because it is not intuitive. Just return error code if the inputed `path` is not legal.

Reply:
Changed